test/test-functions

```diff
@@ -282,7 +282,7 @@ get_ldpath() {
 
 install_missing_libraries() {
     # install possible missing libraries
-    for i in $initdir/{sbin,bin}/* $initdir/lib/systemd/*; do
+    for i in $initdir{,/usr}/{sbin,bin}/* $initdir{,/usr}/lib/systemd/*; do
```

Comment:
I think we should rename `test-functions` to `test-functions.in` and use `@bindir@`, `@sbindir@` and so on.
But this is another big issue :)